pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -115,6 +123,12 @@ func (p *criStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 			continue
 		}
 
+		caStats, found := caInfos[containerID]
+		if !found {
+			glog.Errorf("Unable to find cadvisor stats for %q", containerID)
+			continue
```

Comment:
We may not want to continue. The goal is that if the container runtime doesn't have cadvisor integration, we only provide CRI stats.

Reply:
done